scst-store/amr-data-model.hbs.md

```diff
@@ -0,0 +1,31 @@
+# AMR Data Model
+
+This topic describes the different data models used in the AMR to store artifact information.
+
+## <a id='overview'></a> Overview
+
+Here is a diagram that shows the data models currently used in AMR to store artifact information and the relation between them.
+
+![Diagram of the AMR Data Models](images/AMR-data-model.jpg)
+
+## AMR Data Models
+
+Currently in the AMR there are two types of data models, `Application` and `Location`.
```

Comment:
Change "Application" to App just because we been using app everywhere in our queries